src/Glpi/Inventory/Conf.php

```diff
@@ -384,26 +389,69 @@ public function showConfigForm()
             echo '</div>';
             echo "</td>";
             echo "</tr>";
-
             echo "<tr class='tab_bg_1'>";
             echo "<td>";
-            echo "<label for='auth'>" . __s('Authorization header') . "</label>";
+            echo CommonGLPI::createTabEntry(__s('Authorization header'), 0, null, 'ti ti-cloud-lock');
```

Comment:
I don't think it's relevant to use this method here; we're not creating a tab; plus that may cause issues when one will try to convert this page to a Twig template.

Reply:
It's just that we've done this for all the other options

```
echo \Item_Disk::createTabEntry(\Item_Disk::getTypeName(Session::getPluralNumber()), 0, \Item_Disk::getType());
echo \Software::createTabEntry(\Software::getTypeName(Session::getPluralNumber()), 0, \Software::getType());
echo \Monitor::createTabEntry(\Monitor::getTypeName(Session::getPluralNumber()), 0, \Monitor::getType());
 echo \Printer::createTabEntry(\Printer::getTypeName(Session::getPluralNumber()), 0, \Printer::getType());
....
```

Reply:
However, instead of using `CommonGLPI`, we could use `Oauthclient`.

```php
echo \OAuthClient::createTabEntry(\OAuthClient::getTypeName(Session::getPluralNumber()), 0, \OAuthClient::getType());
```

Reply:
Sounds strange, but OK... That part must be migrated to a twig template anyway.

Reply:
> However, instead of using `CommonGLPI`, we could use `Oauthclient`.
> 
> ```
> echo \OAuthClient::createTabEntry(\OAuthClient::getTypeName(Session::getPluralNumber()), 0, \OAuthClient::getType());
> ```

I do not get the point here, bu I think it's not necessary to change. As I said, this must be migrated to Twig, and therefore won't use `createTabEntry()` anymore.

Also, I do not know what `OAuthClient::getTypeName()` gives, but it should certainly be used only in oauth context, which is not the case here.